### ISSUE TITLE
Button width not respecting width attribute

### ISSUE THREAD

**Issue by ellisonbg** [ipython/ipywidgets#598]
From the mailing list:

```python
bA = Button(description="a description", width=200, color='lime')
bB = Button(description="b description",width=100, color='lime')
display(bA)
display(bB)
```

In the above code, both button-s bA and bB are of same length

How do I make the width of the button appropriate to the length of the description text?

**Comment by ellisonbg** [comment]
I though we got this working, but I do observe the same behavior as this user reports.

**Comment by SylvainCorlay** [comment]
This was fixed in #602
